airbyte-integrations/connectors/source-cockroachdb/src/test-integration/java/io/airbyte/integrations/source/cockroachdb/CockroachDbSourceDatatypeTest.java

```diff
@@ -313,24 +313,12 @@ protected void initTests() {
             .addExpectedValues("a", "abc", "Миші йдуть;", "櫻花分店", "", null, "\\xF0\\x9F\\x9A\\x80")
             .build());
 
-    // JdbcUtils-> DATE_FORMAT is set as ""yyyy-MM-dd'T'HH:mm:ss'Z'"" for both Date and Time types.
-    // Time (04:05:06) would be represented like "1970-01-01T04:05:06Z"
     addDataTypeTestData(
         TestDataHolder.builder()
             .sourceType("time")
             .airbyteType(JsonSchemaType.STRING)
             .addInsertValues("'04:05:06'", null)
-            .addExpectedValues("1970-01-01T04:05:06Z")
-            .addNullExpectedValue()
-            .build());
-
-    // Time (04:05:06) would be represented like "1970-01-01T04:05:06Z"
-    addDataTypeTestData(
-        TestDataHolder.builder()
-            .sourceType("timetz")
```

Comment:
> I think we should keep the test cases for both time and timetz, since those are expected to behave differently? (i.e. eventually, we'll want to `discover` them as `TimeWithTimezone` and `TimeWithoutTimezone` respectively)

rolled it back